### ISSUE TITLE
Inria dataset validation set

### ISSUE THREAD

**Issue by calebrob6** [microsoft/torchgeo#654]
https://project.inria.fr/aerialimagelabeling/contest/ says, "For validation, we suggest to remove the first five images of every location (e.g., austin{1-5}.tif, chicago{1-5}.tif) from the training set."

It would be nice to implement this in our Dataset, then have the option to ignore it or not in the DataModule (i.e. when you want to train with as many labels as possible).
